docker/Dockerfile

```diff
@@ -2,30 +2,18 @@ ARG UBUNTU_VERSION=18.04
 FROM ubuntu:${UBUNTU_VERSION}
 
 # Use bash as the default shell
-RUN ln -snf /bin/bash /bin/sh
+SHELL ["/bin/bash", "-c"]
 
 # Update core packages
 RUN apt-get -y update
-RUN apt-get -y upgrade
-RUN apt-get -y dist-upgrade
 
 # Get some apt packages
 RUN apt-get -y install autoconf clang cmake g++ gcc gfortran git libblas-dev \
                        libhdf5-dev liblapack-dev libpython2.7-dev libtool \
                        python-numpy python-pip python-setuptools wget
 
-# Get astyle
-RUN if [ "${UBUNTU_VERSION}" == "18.04" ]; then \
```

Comment:
Removing these lines will disallow the ability to do the housekeeping run on Ubuntu 16.04. This is fine, as long as the housekeeping run is changed back to being on 18.04.

Reply:
I readied those line for ubuntu 18.04 only as housekeeping is only done there.